cmd/honeytrap/main.go

```diff
@@ -99,11 +100,26 @@ func serve(c *cli.Context) error {
 	var options []server.OptionFn
 
 	if v := c.String("config"); v == "" {
-	} else if fn, err := server.WithConfig(v); err != nil {
+	} else if u, err := url.Parse(v); err != nil {
 		ec := cli.NewExitError(err.Error(), 1)
 		return ec
-	} else {
+	} else if u.Scheme == "" {
```

Comment:
if you support both u.Scheme == "" and u.Scheme == "file", you can use u.Path instead. This allows also urls like file:///etc/honeytrap.config.yml